modules/debts_api/lib/debts_api/v0/financial_status_report_service.rb

```diff
@@ -236,7 +236,7 @@ def validate_form_schema(form)
     end
 
     def send_confirmation_email(template_id)
-      return unless Flipper.enabled?(:fsr_confirmation_email)
+      return unless Rails.env.production?
```

Comment:
Rails.env.production will enable this in dev, staging, sandbox, and prod, not just prod. 
If you are wanting this to run in production only, please use `Settings.vsp_environment == 'production'`

Reply:
That's great to know, thank you!